azure-app-service/massdriver.yaml

```diff
@@ -78,16 +78,40 @@ params:
               const: P3v3
         minimum_worker_count:
           title: Minimum instances
-          description: Minimum number of instances for autoscale.
+          description: Minimum number of instances for autoscale. (Enabling zone balancing multiplies this value by 3)
           type: integer
           minimum: 1
           maximum: 30
         maximum_worker_count:
           title: Maximum instances
-          description: Maximum number of instances for autoscale.
+          description: Maximum number of instances for autoscale.(Enabling zone balancing multiplies this value by 3)
           type: integer
           minimum: 1
           maximum: 30
+      dependencies:
+        location:
+          oneOf:
+            - properties:
+                location:
+                  enum:
+                    - westus2
+                    - westus3
+                    - eastus
+                    - eastus2
+                    - southcentralus
+                    - centralus
+                zone_balancing:
```

Comment:
do you know if this can be changed without recreating the service?

Reply:
Ah, yes, it can't be changed. Not without recreating the app, but a bug within the provider that's properly documented already. Thanks for reminding me to add the immutable field.

Reply:
I got you dood. Do you mind pawing around to see if we have that documented / a ticket for it? If you can't find it just drop something simple in the backlog and we can come back to it.

Reply:
It's a bug with the [AzureRM Terraform provider](https://github.com/hashicorp/terraform-provider-azurerm/issues/18315)

Reply:
That's for the azure function, but I assume it's the same for an app service. I'd say we add immutable and call it good for now.